common/src/main/java/com/wynntils/features/overlays/TradeMarketBulkSellFeature.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.overlays;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.config.RegisterConfig;
+import com.wynntils.core.features.Feature;
+import com.wynntils.handlers.chat.event.ChatMessageReceivedEvent;
+import com.wynntils.mc.event.ContainerSetSlotEvent;
+import com.wynntils.mc.event.ScreenOpenedEvent;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.utils.mc.ComponentUtils;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.wynn.ContainerUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.client.gui.screens.inventory.ContainerScreen;
+import net.minecraft.client.gui.screens.inventory.MenuAccess;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.inventory.ChestMenu;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.OVERLAYS)
+public class TradeMarketBulkSellFeature extends Feature {
+
+    @RegisterConfig
+    public Config<Integer> bulkSell1Amount = new Config<>(64);
+
+    @RegisterConfig
+    public Config<Integer> bulkSell2Amount = new Config<>(1728);
```

Comment:
These numbers seem a bit random. I assume they come from Legacy? (I always thought these numbers were a bit arbitrary, too).

What is the need for a "sell <fixed amount>" button? I understand "Sell all", that is great. But the others? Do we need 3 fixed amount buttons? It seems overkill.

Reply:
Supposedly the reasoning from legacy was that 64 does one stack, 1728 does your inventory, and 6399 does your inventory+bank

Reply:
Ah, that'd explain the numbers at least. Thanks. Hard to imagine anyone filling your entire bank with a single type of item, though :)